app/views/shared/_my_portal_links.html.erb

```diff
@@ -28,7 +28,11 @@
             </li>
           <% end %>
           <% if insured %>
-            <li><%= link_to l10n('.my_insured_portal'), main_app.family_account_path(tab: 'home'), class: 'header-text' %></li>
+            <% if eligible_to_redirect_to_home_page?(current_user) %>
+              <li><%= link_to l10n('.my_insured_portal'), main_app.family_account_path(tab: 'home'), class: 'header-text' %></li>
+            <% else %>
+              <li><%= link_to l10n('.my_insured_portal'), main_app.failed_validation_insured_interactive_identity_verifications_path(person_id: current_user.person.id), class: 'header-text' %></li>
```

Comment:
Need to redirect to the `ridp_agreement_insured_consumer_role_index_path` path and not Failed Validation.